### ISSUE TITLE
Exif returned dataType from parser

### ISSUE THREAD

**Issue by jalilbengoufa** [MikeKovarik/exifr#36]
Hi im wondering if it's normal that the parser return sometimes data as `Uint16Array` instead of a simple number, in this example 
![Kodak_CX7530](https://user-images.githubusercontent.com/27975744/93116269-0e5bb680-f68b-11ea-9938-1a160aadc5b4.jpg)

 we can see it with `ExifImageHeight` and `ExifImageWidth`

Parsed data:

```
ApertureValue: 4.4
ColorSpace: 1
ComponentsConfiguration: Uint8Array(3) [1, 2, 3]
Contrast: "Normal"
CreateDate: Sat Aug 13 2005 09:47:23 GMT-0400 (Eastern Daylight Time) {}
CustomRendered: "Normal"
DateTimeOriginal: Sat Aug 13 2005 09:47:23 GMT-0400 (Eastern Daylight Time) {}
DigitalZoomRatio: 0
ExifImageHeight: Uint16Array(2) [78, 0]
ExifImageWidth: Uint16Array(2) [100, 0]
ExifVersion: "2.2.1"
ExposureCompensation: 0
ExposureIndex: 80
ExposureMode: "Auto"
ExposureProgram: "Normal program"
ExposureTime: 0.004
FNumber: 4.6
FileSource: "Digital Camera"
Flash: "Flash did not fire, auto mode"
FlashpixVersion: "1.0"
FocalLength: 16.8
FocalLengthIn35mmFormat: 102
GPSLatitude: (3) [0, 22.278, 0]
GPSLatitudeRef: "S"
GPSLongitude: (3) [36, 3.385, 0]
GPSLongitudeRef: "E"
GPSVersionID: "2.2.0.0"
GainControl: "None"
LightSource: "Unknown"
Make: "EASTMAN KODAK COMPANY"
MaxApertureValue: 4.4
MeteringMode: "Pattern"
Model: "KODAK CX7530 ZOOM DIGITAL CAMERA"
ModifyDate: Thu Jul 31 2008 10:39:26 GMT-0400 (Eastern Daylight Time) {}
Orientation: "Horizontal (normal)"
ResolutionUnit: "inches"
Saturation: "Normal"
SceneCaptureType: "Standard"
SceneType: "Directly photographed"
SensingMethod: "One-chip color area sensor"
Sharpness: "Normal"
ShutterSpeedValue: 8
Software: "GIMP 2.4.5"
SubjectDistanceRange: "Unknown"
WhiteBalance: "Auto"
XResolution: 72
YCbCrPositioning: 1
YResolution: 72
latitude: -0.37129999999999996
longitude: 36.056416666666664

```

**Comment by MikeKovarik** [comment]
Hello. This is strange but in fact a valid exif. The tag doesnt strictly need to adhere to a number type. And in this case Gimp used number array. It's not expected, but valid... oh well. You can check if against other online exif tools (like http://exif.regex.info/exif.cgi). But I may implement rule that would unpack the value from array 🤔

**Comment by jalilbengoufa** [comment]
yeah that would be great if it's done by exifr.

**Comment by MikeKovarik** [comment]
Fixed in 6.0.0
